packages/api-codec/src/codec/Base.ts

```diff
@@ -2,6 +2,10 @@
 // This software may be modified and distributed under the terms
 // of the ISC license. See the LICENSE file for details.
 
+import logger from '@polkadot/util/logger';
+
+export const l = logger('codec');
```

Comment:
Is this needed?

Reply:
Only for debug logging. I continue adding and removing the bloody logs, so the `l.debug(() => )` ends up as a noop in normal mode and as a bonus when running tests it spits out the result.

Reply:
Currently Struct & createType use it. (Should have added it earlier actually, I continued committing console.log changes.